source/adressering-av-undertegner.rst

```diff
@@ -53,6 +53,8 @@ For private organizations
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 If the sender is a **private organization**, you must yourself enter the email address and/or mobile number of the recipient. This also applies if you address the signer using national identity number. Private organizations cannot use KRR.
 
+If you address signers by their national identity number and they have a Digipost account, the notification will be sent to their Digipost inbox. If they do not have a Digipost account, the notification will be sent by email and/or SMS as described above.
+
```

Comment:
Rett over her står det tittel "For private organizations" så derfor er ikke det med i teksten